src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -70,6 +70,24 @@ function getLandoConfig() {
 	};
 }
 
+const appMap: Map<string, App> = new Map();
+
+async function getLandoApplication( lando: Lando, instancePath: string ): Promise<App> {
+	if ( appMap.has( instancePath ) ) {
+		return Promise.resolve( appMap.get( instancePath ) );
```

Comment:
Since this is an async function wouldn't `return appMap.get( instancePath );` be sufficient?

Reply:
async functions return promises no matter what 🙂

Although a JS engine will convert a "plain" value into a promise, type checkers like Flow and TypeScript complain if we return something else than a promise.

Reply:
> async functions return promises no matter what

right, that's exactly what I meant :) 

> type checkers like Flow and TypeScript complain if we return something else than a promise.

ah I see